### DISCUSSION TITLE
Essentials Support

### DISCUSSION THREAD

**Discussion by DesertRunner76** [winleafs/Winleafs#201] [Q&A]
Hi, 

Sorry if this is the wrong area but I am new to GitHub. I have Nanoleaf Essentials LED Strip and bulb. They are connected to a thread network via the thread border router attached to my Nanoleaf Elements. The thread network operates via the Nanoleaf Android App. I am unable to get the screen mirrioring funtion to work with the essentials via the Official nanoleaf PC app and am wondering if Winleafs is able to /  planning to add this funtionality at some point?

Thanks.

**Comment by StijnOostdam** [comment]
Hey, thanks for the question. We do not own any LED strips, so it is difficult to develop for us. What you could do to help is first try to connect to your strips in Winleafs. If that works, then you need to go to the logs, which you can find in %appdata%. There should be a line that looks like this:

`Sending following request to Nanoleaf: Address: http://192.168.1.148:16021/, URL: api/v1/JME8ADFC1vuFBkQsdfs5465sWT2CkuSu/panelLayout/layout`

Transform that to a full URL:

`http://192.168.1.148:16021/api/v1/JME8ADFC1vuFBkQsdfs5465sWT2CkuSu/panelLayout/layout`

Then go to that URL in your browser and send us the output. Then we have an example to work with and can probably enable drawing and perhaps also screen mirror.

**Reply by DesertRunner76** [reply]
Thanks for the reply!

Neither the Strip or the bulb are available to connect in Winleafs when I go to add a device (they only connect via Thread or Bluetooth). My elements is connected fine however. On an unrelated note, the Winleafs app (Windows 10) crashes when I select the options button.

**Reply by StijnOostdam** [reply]
Hey, that is too bad to hear. In that case we are not planning on adding them to our application. I was hoping that the strip uses existing Shape functionality but that does not seem to be the case.

**Reply by DesertRunner76** [reply]
No worries, appreciate the replies nonetheless :D
